src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -2018,6 +2019,7 @@ namespace nupic {
     }
     NTA_CHECK(check_vector_eq(trueNeighborsMap5, neighborsMap));
 
+    numColumns = 8;
```

Comment:
Perhaps the 30 column and 8 column parts should be different tests? Or you can wrap each block in an anonymous namespace so variables like `numColumns` are defined separately?